### ISSUE TITLE
Dev channel : Scrolling on Webapps (only ?) becomes less fluid

### ISSUE THREAD

**Issue by gbdomubpkm** [ubports/morph-browser#357]
For me, dev channel  BQ  E5 HD, OPO for 2/3 days: it didn't really question me at first but now yes..

This is not reproducible in a systematic way but happens randomly (according to my knowledge and observations): the scrolling becomes chopped, jerky and no longer in a fluid movement.

Examples of observations :

1 Webapp of Github (Webber creation): when viewing the content of a 'long' issue commentary.

2 Webapp 'My UT Forum' ( J M Reyes) : when consulting the content of a 'long' topic.

Closing the webapp and reopening it restores normal scrolling.

I don't remember if I've ever noticed  this jerky scrolling under Morph-browser (to be confirmed).

**Comment by gbdomubpkm** [comment]
BQ E5 HD : same problem quite often by consulting dekko emails

**Comment by gbdomubpkm** [comment]
BQ E5 HD last on last rc channel. The problem is still very present and quite annoying. You surely could reproduce it with your BQ 4.5 I think. What is the nature of the problem? Lack of memory? Anything else? Corrected by a pending pull request?
If it helps, what are the useful logs to communicate to you at the time of the problems?

**Comment by mateosalta** [comment]
maybe the way certain sites load. there are a few flags we can set that involve smooth scrolling - and the way the display is loaded, it is hard to notice a difference on most phones, perhaps the devices like the e5 would be good to test?

**Comment by gbdomubpkm** [comment]
Not only webapp, even with morph-browser and  on the Vollaphone !!
For example today, RC 8 by consulting https://ubports.com/blog/ubport-blogs-news-1/post/ubuntu-touch-q-a-94-3741.
But the problem occured before, RC 7 or devel. With the RAM of Volla, even if only a small part of it is allocated, more than surprising!!
Close morph and reload the link solve the problem.

**Comment by gbdomubpkm** [comment]
The problem is there under the Volla  RC 20 with Morph and webapps.
Is it because there is no hardware acceleration that would allow a better and faster loading of web pages?

I have noticed that the problem is very rare when we wait a certain amount of time and let the page load completely before scrolling (favicon displayed).
On the other hand, if the web page is not fully loaded, it necessarily causes jerky scrolling all the time.

**Comment by gbdomubpkm** [comment]
On the Volla OTA-19,  scrolling work like a charm with [Onion Surf](https://open-store.io/app/onion-surf.collaproductions) based on morph-browser whereas the problem is always there on morph-browser  ! 🤔

**Comment by mateosalta** [comment]
oh maybe is related to the scrollbar css injection?

**Comment by gbdomubpkm** [comment]
Yes the scrolling on Onion does not freeze even when pages are not loaded, maybe you have tested it.  I asked the question (whether something is different) to try to move the problem forward.

**Comment by mateosalta** [comment]
one of the changes was that he removed the scrollbar css injection i think - perhaps that is part of the problem?

**Comment by gbdomubpkm** [comment]
Hello @balcy
Onion's scrolling  seems smoother than morph-browser's scrolling.. Do you think something could be improved quite easily about that in morph ?

**Comment by lduboeuf** [comment]
humm haven't noticed scrolling issue on N5. Is this device specific ?

**Comment by gbdomubpkm** [comment]
The answer of @nicolascolla to my question [here](https://github.com/nicolascolla/onion-surf/issues/8) : "the reason for the smoother scrolling seems to be very simple: In order to have a fancy scrollbar, Morph injects javascript into a page once it has loaded (you can see the scrollbar change while loading if you pay attention). I have removed it because it makes it easier to scroll by dragging it when you are having scrolling issues (which a lot of UT users have) and it seems like a nice side effect is better scrolling''.

Are there command lines to easily test this on morph-browser ?
